index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <Style Type="text/css"></Style>
+    </head>
+    <TITLE>Vincent John Scalco, III</TITLE>
+    <body>
+            <div id="nav">
+        <ul>
+            <li><a href="#Welcome!">Welcome!</a></li>
+            <li><a href="#About Me">About Me</a></li>
+            <li><a href="#Projects">Projects</a></li>
+            <li><a href="#Contact">Contact</a></li>
+        </ul>
+        </div>
+    </body>
+
+</html>
+<html lang="en">
+<header>
+<link rel="stylesheet" href="./styles.css">
+<script defer="defer" src="./components/tabs.js"></script>
+</header>
+    <div class="name">
+    <h1>Vincent John Scalco, III</h1>
+</div>
+
+<div id="Welcome!">
+<h1 class="container">Welcome!</h1>
+<p class="p"></p>
+</div>
+
+<div id="About Me">
+
+<h1 class="container">About Me</h1>
+<img src=""" width="275" height="265" />
```

Comment:
It's bad practice to do inline styling like this, especially since you have css files. Definitely move all of these into the css files.